### ISSUE TITLE
  Different evaluation methods?

### ISSUE THREAD

**Issue by stidk** [open-mmlab/mmdetection3d#1138]
Why do my  running results of pedestrian with pointpillars in mmdet3d are so high compare the ranking of Kitti datasets.

**Comment by ZCMax** [comment]
Could you should your detailed running results?

**Comment by stidk** [comment]
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:63.8746, 60.3893, 56.6835
bev  AP:59.6675, 54.1145, 48.7748
3d   AP:51.5823, 46.4137, 41.9303
aos  AP:47.28, 45.14, 42.49
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:63.8746, 60.3893, 56.6835
bev  AP:70.8316, 68.4001, 64.3143
3d   AP:70.7208, 68.1585, 63.9440
aos  AP:47.28, 45.14, 42.49
Cyclist AP@0.50, 0.50, 0.50:
bbox AP:87.6756, 71.4933, 68.9722
bev  AP:84.7058, 67.9081, 63.6475
3d   AP:82.7176, 63.2353, 60.2419
aos  AP:86.29, 67.61, 65.26
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:87.6756, 71.4933, 68.9722
bev  AP:87.1581, 70.0334, 68.0850
3d   AP:87.1563, 69.9335, 67.6814
aos  AP:86.29, 67.61, 65.26
Car AP@0.70, 0.70, 0.70:
bbox AP:90.6583, 89.2307, 86.8346
bev  AP:89.8226, 86.9126, 79.6998
3d   AP:83.9184, 75.7401, 68.5678
aos  AP:90.57, 88.68, 85.97
Car AP@0.70, 0.50, 0.50:
bbox AP:90.6583, 89.2307, 86.8346
bev  AP:90.7174, 89.9463, 89.2889
3d   AP:90.7174, 89.8197, 89.0300
aos  AP:90.57, 88.68, 85.97

Overall AP@easy, moderate, hard:
bbox AP:80.7362, 73.7044, 70.8301
bev  AP:78.0653, 69.6451, 64.0407
3d   AP:72.7394, 61.7964, 56.9133
aos  AP:74.72, 67.14, 64.57

**Comment by ZCMax** [comment]
Does ranking of Kitti datasets means the kitti leaderboard ranking? It reports the test datasets results on kitti, while the results provided by you is the val datasets results.Besides, different hyper paramters and data augmentaions will result in different performance.

**Comment by Tai-Wang** [comment]
The performance of cyclists and pedestrians can drop a lot (~10 AP) on the test set (compared to the validation set) due to the small size of the training set. This problem can be alleviated significantly on large-scale datasets.
